boards/saml21-xpro/include/periph_conf.h

```diff
@@ -38,14 +38,22 @@ extern "C" {
  * @name    Timer peripheral configuration
  * @{
  */
-#define TIMER_NUMOF        (1U)
-#define TIMER_0_EN         1
+static const tc32_conf_t timer_config[] = {
+    {   /* Timer 0 - System Clock */
+        .dev            = TC0,
```

Comment:
Here a:
`        .flags          = TC_CTRLA_MODE_COUNT32,`
Is missing.

Reply:
oops, just double checked but it seems this was the only board where I missed that.

Reply:
Thanks for fixing!